modules/ui.js

```diff
@@ -7,6 +7,8 @@ const localTime = document.querySelector('.date-time');
 const listPage = document.querySelector('.book-list-page');
 const AddPage = document.querySelector('.add-book');
 const contactPage = document.querySelector('.contact');
-const navBar = document.querySelectorAll('.item-link');
```

Comment:
- [ ] When your website loads then the default page is Contact Page. Please make List your default page :blush: